analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -144,14 +135,16 @@ class GoMod(
     }
 
     private fun getVendorModules(projectDir: File): Set<Identifier> {
-        val vendorModulesJson = run("list", "-json", "-m", "-mod=vendor", "all", workingDir = projectDir)
+        return run(projectDir, "mod", "vendor", "-v")
             .requireSuccess()
-            .stdout
-
-        return jsonMapper
-            .readValues<ModuleDependency>(JsonFactory().createParser(vendorModulesJson))
-            .asSequence()
-            .mapTo(mutableSetOf()) { Identifier(managerName, "", it.name, it.version.orEmpty()) }
+            .stderr
+            .lineSequence()
+            .filter { it.startsWith("# ") }
```

Comment:
Consider moving the filter condition into the `map` block as it is closely related to the `removePrefix` call.

Reply:
This would require to use `mapNotNull` instead and add one level of indentation to the block for the `if` condition. Using `filter` is nicer to read and better matches the function style in my opinion. And the condition is still very close to the `removePrefix`.